crates/miden-lib/asm/account_components/multisig_rpo_falcon_512.masm

```diff
@@ -293,7 +298,87 @@ export.auth_tx_rpo_falcon512_multisig
 
     # ------ Checking threshold is >= num_verified_signatures ------
 
+    # 1. initialize transaction_threshold = 0
+    # 2. iterate through all account procedures
+    #   a. check if the procedure was called during the transaction
+    #   b. if called, check if this procedure was included in the proc_threshold_map
+    #   c. if included, get its proc_threshold
+    #   d. if proc_threshold > transaction_threshold, set transaction_threshold = proc_threshold
```

Comment:
Could we move this logic into a helper procedure? Maybe something like `get_transaction_threshold`? Or does this complicate things for some reason?

Reply:
also this addresses @partylikeits1983's comment, I believe?
> potentially using local memory and or breaking the procedure down into smaller helper procedures.